src/woo_publications/publications/api/serializers.py

```diff
@@ -11,20 +16,25 @@
 from ..models import Document, Publication, Topic
 
 
-class EigenaarSerializer(serializers.Serializer):
+class EigenaarSerializer(serializers.ModelSerializer[OrganisationMember]):
     weergave_naam = serializers.CharField(
-        source="display_name",
-        read_only=True,
+        source="naam",
         help_text=_("The display name of the user, as recorded in the audit trails."),
     )
     identifier = serializers.CharField(
```

Comment:
I think we can remove this field declaration entirely, DRF will take it from the model. Probably good to move the help text definition to the model too?